### ISSUE TITLE
`cxflow resume` fails if `model.n_gpus` is less than the original value

### ISSUE THREAD

**Issue by blazekadam** [Cognexa/cxflow-tensorflow#66]
This may be related to batch normalization routines.
